app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :posts, foreign_key: :post_id
```

Comment:
- [ ] Here it should be `belongs_to :post`, there is no need to define ` foreign_key`.